Multisig/Logic/Models/Transaction.swift

```diff
@@ -42,4 +42,30 @@ extension Transaction {
         nonce = multiSigTxInfo.nonce
         safeTxHash = multiSigTxInfo.safeTxHash
     }
+
+    var safeEncodedTxData: Data {
+        [
+            Safe.DefaultEIP712SafeAppTxTypeHash,
+            to.data32,
+            value.data32,
+            EthHasher.hash(data.data),
+            operation.data32,
+            safeTxGas.data32,
+            baseGas.data32,
+            gasPrice.data32,
+            gasToken.data32,
+            refundReceiver.data32,
+            nonce.data32
+        ]
+        .reduce(Data()) { $0 + $1 }
+    }
+
+    func encodeTransactionData(for safe: AddressString) -> Data {
+        return [
+            Data([0x19]),
+            Data([0x01]),
```

Comment:
please extract magic numbers in named constants

Reply:
I don't see a big additional value in it. Probably making a comment with a link to our contract would make more sense here, wdyt?

Reply:
please don't just use the magic constants, they are not obvious

Reply:
done